### PR TITLE
LVM-482 Implement Student Comments

### PR DESCRIPTION
**Pull request by vcantu**
[LVM-482](https://cs5500-jira.ccs.neu.edu/browse/LVM-482)

Adds student comments just like tutor comments

### CODE REVIEW

File: app/controllers/student_comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+class StudentCommentsController < ApplicationController
+  before_action :set_student_comment, only: [:edit, :update, :destroy]
+
+  def new
+    @student_comment = StudentComment.new
+    @student = Student.of(current_user).find(params[:student])
```
Comment (zack) [inline app/controllers/student_comments_controller.rb:6]: i like to use model_id as the param, but not a big deal.

**Review by zack — APPROVED** [review]
Nice! We'll want to add authors to these though. We can do that in another PR though. Please make a ticket for that?

**Comment by zack** [comment]
Also, you need to rebase.

File: app/views/tutor_comments/new.html.erb [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <div class="col-md-12">
       <%= link_to tutor_path(@tutor) do %>
         <span class="glyphicon glyphicon-arrow-left"></span>
-        <span class="h4">Back</span>
+        <span class="h4">Back To Tutor</span>
```
Comment (xingyif) [inline app/views/tutor_comments/new.html.erb:6]: Choose one:
`Back to Tutors`
or 
`Back to All Tutors`

Comment (zack) [reply app/views/tutor_comments/new.html.erb:6]: This link goes back to _a tutor_. The copy is good. (look at line 4)

**Pull request merged**